package.json

```
@@ -2,6 +2,15 @@
   "name": "broadcast-channel",
   "version": "4.0.0",
   "description": "A BroadcastChannel that works in New Browsers, Old Browsers, WebWorkers and NodeJs",
+  "exports": {
+    "node": {
+      "import": "./dist/es/index.js",
+      "default": "./dist/es5node/index.js"
+    },
+    "browser": {
+      "default": "./dist/lib/index.es5.js"
```

I would also add `import` for the browser as ESM modules can be used in the browser

```suggestion
      "import": "./dist/es/index.js",
      "default": "./dist/lib/index.es5.js"
```

---

I think the ESM module is supposed to be working in the browser but webpack somehow doesn't eliminate this line:
```
import * as NodeMethod from '../../src/methods/node.js';
```
from the ESM module. So if `import` is added here, I run into the compilation error again because of all the node native module imported inside `../../src/methods/node.js`

---

Is it possible to have two ESM builds, one for the browser and one for node. And then with the `exports` syntax, simply add
```
"import": "./dist/es-browser/index.js",
```